recipes/bircher/meta.yaml

```diff
@@ -0,0 +1,50 @@
+{% set name = "birka" %}
+{% set version = "0.1.0rc6" %}
```

Comment:
This one falls under https://github.com/conda-forge/cfep/blob/main/cfep-05.md

Reply:
Updated to 0.1.0